doc/api/http.md

```diff
@@ -3661,13 +3661,18 @@ Passing an `AbortSignal` and then calling `abort` on the corresponding
 `AbortController` will behave the same way as calling `.destroy()` on the
 request itself.
 
-## `http.validateHeaderName(name)`
+## `http.validateHeaderName(name[, label])`
 
 <!-- YAML
 added: v14.3.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/46143
+    description: The `label` parameter is added.
 -->
 
 * `name` {string}
+* `label` {string} Label for error message. **Default:** `Header name`.
```

Comment:
```suggestion
* `label` {string} Label for error message. **Default:** `'Header name'`.
```

Reply:
Thank you for correction. Applied your suggestion.